2019-04-29 12:41:30,272 DEBUG   MainThread:19310 [wandb_config.py:_load_defaults():81] no defaults not found in config-defaults.yaml
2019-04-29 12:41:30,278 DEBUG   MainThread:19310 [cmd.py:execute():722] Popen(['git', 'cat-file', '--batch-check'], cwd=/Users/Caetius/Desktop/DL_Final/DAE, universal_newlines=False, shell=None)
2019-04-29 12:41:30,294 DEBUG   MainThread:19310 [cmd.py:execute():722] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=/Users/Caetius/Desktop/DL_Final/DAE, universal_newlines=False, shell=None)
2019-04-29 12:41:30,320 DEBUG   MainThread:19310 [run_manager.py:__init__():452] Initialized sync for le-project/e7romswt
2019-04-29 12:41:30,324 INFO    MainThread:19310 [run_manager.py:wrap_existing_process():986] wrapping existing process 19300
2019-04-29 12:41:30,332 DEBUG   MainThread:19310 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): pypi.org:443
2019-04-29 12:41:30,381 DEBUG   MainThread:19310 [connectionpool.py:_make_request():393] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 31060
2019-04-29 12:41:30,391 INFO    MainThread:19310 [run_manager.py:init_run():810] system metrics and metadata threads started
2019-04-29 12:41:30,391 INFO    MainThread:19310 [run_manager.py:init_run():844] upserting run before process can begin, waiting at most 10 seconds
2019-04-29 12:41:30,403 DEBUG   Thread-13 :19310 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2019-04-29 12:41:30,510 DEBUG   Thread-13 :19310 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 352
2019-04-29 12:41:30,514 INFO    Thread-13 :19310 [run_manager.py:_upsert_run():900] saving patches
2019-04-29 12:41:30,514 DEBUG   Thread-13 :19310 [cmd.py:execute():722] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=/Users/Caetius/Desktop/DL_Final/DAE, universal_newlines=False, shell=None)
2019-04-29 12:41:30,527 DEBUG   Thread-13 :19310 [cmd.py:execute():722] Popen(['git', 'diff', '--cached', '--abbrev=40', '--full-index', '--raw'], cwd=/Users/Caetius/Desktop/DL_Final/DAE, universal_newlines=False, shell=None)
2019-04-29 12:41:30,542 DEBUG   Thread-13 :19310 [cmd.py:execute():722] Popen(['git', 'diff', '--abbrev=40', '--full-index', '--raw'], cwd=/Users/Caetius/Desktop/DL_Final/DAE, universal_newlines=False, shell=None)
2019-04-29 12:41:30,556 DEBUG   Thread-13 :19310 [cmd.py:execute():722] Popen(['git', 'version'], cwd=/Users/Caetius/Desktop/DL_Final/DAE, universal_newlines=False, shell=None)
2019-04-29 12:41:30,590 INFO    Thread-13 :19310 [run_manager.py:_upsert_run():902] saving pip packages
2019-04-29 12:41:30,591 INFO    Thread-13 :19310 [run_manager.py:_upsert_run():904] initializing streaming files api
2019-04-29 12:41:30,592 INFO    Thread-13 :19310 [run_manager.py:_upsert_run():911] unblocking file change observer, beginning sync with W&B servers
2019-04-29 12:41:30,592 INFO    MainThread:19310 [run_manager.py:wrap_existing_process():1003] informing user process we are ready to proceed
2019-04-29 12:41:30,592 INFO    MainThread:19310 [run_manager.py:_sync_etc():1059] entering loop for messages from user process
2019-04-29 12:41:31,280 INFO    Thread-2  :19310 [run_manager.py:_on_file_created():576] file/dir created: /Users/Caetius/Desktop/DL_Final/DAE/wandb/run-20190429_164129-e7romswt/wandb-metadata.json
2019-04-29 12:41:31,281 INFO    Thread-2  :19310 [run_manager.py:_on_file_created():576] file/dir created: /Users/Caetius/Desktop/DL_Final/DAE/wandb/run-20190429_164129-e7romswt/diff.patch
2019-04-29 12:41:31,281 INFO    Thread-2  :19310 [run_manager.py:_on_file_created():576] file/dir created: /Users/Caetius/Desktop/DL_Final/DAE/wandb/run-20190429_164129-e7romswt/config.yaml
2019-04-29 12:41:31,290 DEBUG   Thread-2  :19310 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2019-04-29 12:41:31,388 DEBUG   Thread-2  :19310 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 629
2019-04-29 12:41:31,390 INFO    Thread-2  :19310 [run_manager.py:_on_file_created():576] file/dir created: /Users/Caetius/Desktop/DL_Final/DAE/wandb/run-20190429_164129-e7romswt/requirements.txt
2019-04-29 12:41:31,395 DEBUG   Thread-14 :19310 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2019-04-29 12:41:31,472 DEBUG   Thread-14 :19310 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 743
2019-04-29 12:41:31,478 DEBUG   Thread-14 :19310 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): storage.googleapis.com:443
2019-04-29 12:41:31,778 DEBUG   Thread-14 :19310 [connectionpool.py:_make_request():393] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/dlc423/le-project/e7romswt/config.yaml?Expires=1556556151&GoogleAccessId=gorilla-cloud-storage%40wandb-production.iam.gserviceaccount.com&Signature=HW8FtP3cBeCXe4Dx5JmDGaswSgDLDPkYestYt%2FGjFQRbM00Fqcwac4eKvGxGLnGrSxrDTvSsitTD%2F7w%2F3DzILAosw2aGLBFLTgRw23chxKp8vQB%2FH4x5PicaPyqA3PZIJPTNp6gdQtCwfU4b3Xfn4HHx4qheNYlIp4iFiSM5SQragfG3QS8sVPnyLmrCONzkSSUC%2BeTsupIQxQD385K8YmEEAGb2pvm2i16bQfksdYmljU%2FQC9yipl9REySOxVGPRuSLHFldWDdCSbRYKeoE0fRrp87U9BV5W7%2B2qkOHBvviPaJf1oXIFGUxY0%2FpfBhRzSWiyC5RqNtHN203CR1DfA%3D%3D HTTP/1.1" 200 0
2019-04-29 12:41:32,609 DEBUG   Thread-6  :19310 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2019-04-29 12:41:32,710 DEBUG   Thread-6  :19310 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /dlc423/le-project/e7romswt/file_stream HTTP/1.1" 200 312
2019-04-29 12:41:36,658 DEBUG   Thread-6  :19310 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /dlc423/le-project/e7romswt/file_stream HTTP/1.1" 200 312
2019-04-29 12:41:37,296 INFO    Thread-2  :19310 [run_manager.py:_on_file_created():576] file/dir created: /Users/Caetius/Desktop/DL_Final/DAE/wandb/run-20190429_164129-e7romswt/wandb-history.jsonl
2019-04-29 12:41:37,524 INFO    MainThread:19310 [run_manager.py:_sync_etc():1110] process received interrupt signal, shutting down
2019-04-29 12:41:37,524 INFO    MainThread:19310 [run_manager.py:_sync_etc():1163] closing log streams and sending exitcode to W&B
2019-04-29 12:41:37,525 INFO    MainThread:19310 [run_manager.py:shutdown():918] shutting down system stats and metadata service
9310 [e7romswt:run_manager.py:_sync_etc():1163] closing log streams and sending exitcode to W&B
2019-04-29 12:41:37,525 INFO    MainThread:19310 [e7romswt:run_manager.py:shutdown():918] shutting down system stats and metadata service
